docs(kilo-docs): switch Auto Balanced docs to GPT 5.3 Codex (#8682)
Kilo-Org/kilocode: packages/kilo-docs/pages/code-with-ai/agents/auto-model.md

2. Start working in any mode (Code, Architect, Debug, etc.)
3. The system automatically routes your requests to the best model for that task

> That's it. No configuration needed. The extension shows which model was used for each request — including cost information — when you expand a message. For the exact model mappings for each tier, see the [Gateway docs](/docs/gateway/models-and-providers#kilo-autofrontier).

Author: ```suggestion
That's it. No configuration needed. 

You can see which underlying models are used, as well as the cost, in the expanded model picker. Mapping information is also available on the [Gateway Model page](/docs/gateway/models-and-providers#kilo-autofrontier).
```

2. Start working in any mode (Code, Architect, Debug, etc.)
3. The system automatically routes your requests to the best model for that task

That's it. No configuration needed. 

> You can see which underlying models are used, as well as the cost, in the expanded model picker. Mapping information is also available on the [Gateway Model page](/docs/gateway/models-and-providers#kilo-autofrontier).

Author: ```suggestion
You can see which underlying models are used, as well as the cost, in the expanded model picker. Model mapping information is also available on the [Gateway Model page](/docs/gateway/models-and-providers#kilo-autofrontier).
```

## Tiers

- **Frontier** — Routes to the latest and most capable paid models. Uses different models for reasoning-heavy tasks (planning, architecture, debugging) versus implementation tasks (coding, building, exploring), pairing the right capability to each type of work.
- **Balanced** — Follows the same mode-based routing structure as Frontier but uses a more cost-effective model across all modes. A good default for most developers who want strong AI assistance without paying frontier prices.
- **Free** — Routes to the best available free model on OpenRouter. Because free model availability shifts over time as providers change promotional periods, the mapping is updated server-side — you always get the best free option without having to track what's currently available. Quality will be lower than paid tiers, and the model may change over time.

## Benefits

> ### No Configuration Required

Author: ```suggestion

### Cost Optimization

Automatically uses the best model for a given task, selecting the best balance of cost and capability for a given task. Uses the more economical models for more straight forward tasks, while reserving stronger reasoning models for planning tasks. You get optimal cost-to-capability ratio without thinking about it.

### No Configuration Required
```

## Auto Balanced

`kilo-auto/balanced` follows the same mode-based routing structure as Frontier but uses a more cost-effective model — GPT 5.3 Codex across all modes.

### Mode-to-Model Mapping

| Mode           | Model Used    | Best For                     |
| -------------- | ------------- | ---------------------------- |
| `architect`    | GPT 5.3 Codex | System design, planning      |
| `orchestrator` | GPT 5.3 Codex | Multi-step task coordination |
| `ask`          | GPT 5.3 Codex | Questions, explanations      |
| `plan`         | GPT 5.3 Codex | Planning, reasoning          |
| `general`      | GPT 5.3 Codex | General assistance           |
| `debug`        | GPT 5.3 Codex | Debugging and fixing issues  |
| `code`         | GPT 5.3 Codex | Writing and editing code     |
| `build`        | GPT 5.3 Codex | Implementation tasks         |
| `explore`      | GPT 5.3 Codex | Codebase exploration         |

> **All tasks** use GPT 5.3 Codex, providing strong coding and reasoning performance across all modes at a lower cost than frontier routing.

Author: ```suggestion
**All tasks** use GPT 5.3 Codex (Low), providing strong coding and reasoning performance across all modes at a lower cost than frontier routing.
```
